### PR TITLE
Initial commit of GEOSPLIT

### PR DESCRIPTION
**Pull request by hbs**

### CODE REVIEW

File: warp10/src/main/java/io/warp10/script/functions/GEOSPLIT.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+//
+
+package io.warp10.script.functions;
+
+import io.warp10.continuum.gts.GTSDecoder;
+import io.warp10.continuum.gts.GTSWrapperHelper;
+import io.warp10.continuum.store.thrift.data.GTSWrapper;
+import io.warp10.crypto.OrderPreservingBase64;
```
Comment (ftence) [inline warp10/src/main/java/io/warp10/script/functions/GEOSPLIT.java]: Useless.

File: warp10/src/main/java/io/warp10/script/functions/GEOSPLIT.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+import io.warp10.script.NamedWarpScriptFunction;
+import io.warp10.script.WarpScriptException;
+import io.warp10.script.WarpScriptStack;
+import io.warp10.script.WarpScriptStackFunction;
+
+import org.apache.thrift.TDeserializer;
+import org.apache.thrift.TException;
+import org.apache.thrift.protocol.TCompactProtocol;
```
Comment (ftence) [inline warp10/src/main/java/io/warp10/script/functions/GEOSPLIT.java]: Useless.

File: warp10/src/main/java/io/warp10/script/functions/GEOSPLIT.java [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+import com.geoxp.GeoXPLib;
+import com.geoxp.GeoXPLib.GeoXPShape;
+import com.geoxp.geo.Coverage;
+import com.geoxp.geo.CoverageHelper;
+import com.geoxp.geo.HHCodeHelper;
+
+import java.nio.charset.StandardCharsets;
```
Comment (ftence) [inline warp10/src/main/java/io/warp10/script/functions/GEOSPLIT.java]: Useless.

**Comment by hbs** [comment]
Reason in terms of defining a polygon around each area, with coordinates within [-180,180] and [-90,90] not doable otherwise.

**Comment by pi-r-p** [comment]
OK. `->GEOJSON` handle shapes around IDL very well with a multipolygon, and GEOSPLIT uses the same logic behind. 
It just needs a good documentation.

**Pull request merged**